src/server/transaction.cc

```diff
@@ -794,9 +803,6 @@ void Transaction::UnlockMulti() {
   unsigned shard_journals_cnt =
       ServerState::tlocal()->journal() ? CalcMultiNumOfShardJournals() : 0;
 
-  uint32_t prev = run_count_.fetch_add(shard_data_.size(), memory_order_relaxed);
```

Comment:
what happens here?

Reply:
We don't need it https://github.com/dragonflydb/dragonfly/pull/2455#discussion_r1461804316. We never use this value again, except for CHECKs